website/src/pages/generator.js

```diff
@@ -26,8 +26,13 @@ function Generator() {
 	const dispatch = useDispatch()
 
 	const onFieldChange = (field, e) => {
+        let isBool = false
+        const idArr = e.target.id.split('.')
+        if(idArr[0] === 'true' || idArr[0] === 'false'){
```

Comment:
?

Reply:
there was a bug where if we select true or false it would come enclosed in quotes, like "false" to overcome that issue, this checks if the id includes `InputBoolean`  ids and updated true or false without quote